ui/shared/components/panel/variants/Pathogenicity.jsx

```diff
@@ -117,6 +132,14 @@ const Pathogenicity = React.memo(({ variant, showHgmd }) => {
       <b>{`${title}:`}</b>
       <HorizontalSpacer width={5} />
       <PathogenicityLink {...linkProps} />
+      {
+        title === 'ClinVar' && (
```

Comment:
I think making this conditional on a magic string is not great. I think it would be best to do it based on the presence of `linkProps.submissions` but if we really want it to be based on title we should at least make that a constant

Reply:
there's no reason not to use `linkProps.submissions`, I'll do that instead